logos/benchmarks/benchmark_logos.py

```
@@ -0,0 +1,1240 @@
+            ssh_key=ssh_key,
+            poll_interval_ms=args.poll_interval_ms,
+        )
+    else:
+        indices = args.gpu_indices if args.gpu_indices is not None else [0]
+        print(f"GPU      : local NVML  indices={indices}")
+        tracker = GPUTracker(indices, args.poll_interval_ms)
+
+    tracker.start()
+
+    # ── Run ───────────────────────────────────────────────────────────────
+    print(f"\nRunning...")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)